### ISSUE TITLE
Parsing error: unexpected closing tag with style

### ISSUE THREAD

**Issue by cpboyd** [mdx-js/eslint-mdx#139]
### Subject of the issue

Adding `style={{}}` seems to break parsing of closing tags.
```
error  Parsing error: Unexpected closing tag "Button". It may happen when the tag has already been closed by another tag. For more info see https://www.w3.org/TR/html5/syntax.html#closing-elements-that-have-implied-end-tags
```

### Your environment

*   **OS**: macOS 10.15.2
*   **Packages**:
```
    "@typescript-eslint/eslint-plugin": "^2.17.0",
    "@typescript-eslint/parser": "^2.17.0",
    "eslint": "^6.8.0",
    "eslint-config-prettier": "^6.9.0",
    "eslint-config-react-app": "^5.1.0",
    "eslint-plugin-flowtype": "^4.6.0",
    "eslint-plugin-graphql": "^3.1.1",
    "eslint-plugin-import": "^2.20.0",
    "eslint-plugin-jsx-a11y": "^6.2.3",
    "eslint-plugin-mdx": "^1.6.5",
    "eslint-plugin-prettier": "^3.1.2",
    "eslint-plugin-react": "^7.18.0",
    "eslint-plugin-react-hooks": "^2.3.0",
```
*   **Env**: yarn 1.21.1

### Steps to reproduce

```jsx
---
title: Contact
---

import { Button } from '@rmwc/button';

# Contact

<Button
  raised
  style={{ margin: '8px' }}
>
  Message Me
</Button>
```

### Expected behaviour

The parser should be able to style on a JSX tag.

### Actual behaviour

The parser throws an error.

**Comment by cpboyd** [comment]
Seems to actually be an issue with prettier: https://github.com/prettier/prettier/issues/7041
